### PR TITLE
Docs overhaul - Introduction+Quickstart

### PR DESCRIPTION
**Pull request by HowieG**
Resolves ENG-246

### CODE REVIEW

File: docs/v1/details/environment-variables.mdx [hunk 1]
```diff
@@ -1,19 +1,24 @@
 ---
 title: "Environment Variables"
 description: "Environment Variables that AgentOps will look for"
+mode: "wide"
 ---
 
 ### Sample .env file
 
-```.env
+```python .env
```
Comment (braelyn-ai) [inline docs/v1/details/environment-variables.mdx:9]: is this intentional? its not a python code block

File: docs/v1/examples.mdx [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
 title: 'Examples'
 description: 'Sample code that you can run!'
+mode: "wide"
 ---
 
 ## Explore our example code to see AgentOps in action!
 
 <CardGroup cols={2}>
     <Card title="Basic Example" icon="square-code" href="https://github.com/AgentOps-AI/agentops-py/blob/main/examples/openai-gpt.ipynb">
-        Juypter Notebook with sample code that you can run! Start here!
```
Comment (braelyn-ai) [inline docs/v1/examples.mdx:10]: What's the difference between these?

File: docs/v1/introduction.mdx [hunk 3]
```diff
@@ -1,23 +1,60 @@
...
-</CardGroup>
+# AgentOps is your new terminal
 
-## Integrate with your agent
+<Card title="Terminals suck" icon="trash" iconType="solid" color="#CC0000">
+	- • Lose track of what your agents did in between executions
+	- • Parsing through terminal output searching for LLM completions
+	- • Printing "tool called"
+</Card>
+<Card title="AgentOps doesn't" icon="face-smile-halo" iconType="duotone" color="#00CC00">
```
Comment (braelyn-ai) [inline docs/v1/introduction.mdx:16]: "AgentOps doesn't" is confusing to me. I feel like "AgentOps doesn't suck" is better

**Review by braelyn-ai — APPROVED** [review]

File: docs/v1/details/environment-variables.mdx (hunk 1 above)
Comment (HowieG) [reply docs/v1/details/environment-variables.mdx:9]: Yes it's just to give it highlighting. Was just a black/white blob before. End user doesn't see anything about python

File: docs/v1/examples.mdx (hunk 2 above)
Comment (HowieG) [reply docs/v1/examples.mdx:10]: spelling typo

Comment (HowieG) [reply docs/v1/examples.mdx:10]: spelling typo

File: docs/v1/introduction.mdx (hunk 3 above)
Comment (HowieG) [reply docs/v1/introduction.mdx:16]: Screenshot 2024-05-23 at 4 33 55 PM

Comment (HowieG) [reply docs/v1/introduction.mdx:16]: I think visually it's pretty clear

**Pull request merged**